### ISSUE TITLE
dataset partitioning

### ISSUE THREAD

**Issue by CleanSeaSalt** [AIR-THU/DAIR-V2X-Seq#25]
Hello, I would like to ask how this sequence data set is divided into training set, verification set and test set?

**Comment by jjrCN** [comment]
same question

**Comment by jjrCN** [comment]
@haibao-yu

**Comment by AcoolHusky** [comment]
It's said the split is train/val/test=5:2:3 in the paper, but I only find the train and val part in the official dataset...
